docs/intro.rst

```diff
@@ -238,6 +238,42 @@ can be installed.
 .. _PyPI: https://pypi.python.org/pypi
 
 
+.. _`Building a compiled archive with Cython`:
+
+Building a compiled archive with Cython
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+The 'Cython' package allows building a wheel distribution archive containing
+compiled code for Python packages such as pywbem. These distribution archives
+are nicknamed "cythonized" wheel distribution archives. They have the wheel
+format, but are specific to a Python version and to the target operating system.
+
+Since the code in a cythonized wheel archive is native machine code for the
+target system, this speeds up execution in compute-intensive areas of pywbem
+such as when parsing the CIM-XML responses of a WBEM server. The downside is
+that a cythonized wheel archive is significantly larger than a universal wheel
+archive, and the memory consumption is also larger.
```

Comment:
propose change to "both the wheel archive  and memory consumption are significantly  larger that a universal wheel archive."

Reply:
Well, I did not measure the memory consumption, and while I think it must be larger based on the archive size, I would not necessarily call this out to be significant without having measured it. So I would like to have two separate comparison statements. NOT CHANGED.

Reply:
In the meeting today, Karl agreed to go without resolving this comment.